src/tools/vs-code.md

```diff
@@ -25,12 +25,12 @@ Updates to the extensions are shipped on a regular basis.
 By default, VS Code automatically updates extensions when
 updates are available.
 
-To install updates manually:
+To install updates yourself:
```

Comment:
I'm curious. Why is "yourself" better than "manually"? It seems to me that manually is more precise.

Reply:
I find "automatically" and "manually" unnecessary. A user can't do anything automatically and no application or process can do something manually. Using "yourself" makes it clear who is performing the action. So, it's a double GSG win: active voice and remove extraneous words.